changed side nav, fixed double scroll bars

mentos1386 opened the pull request:
Fix: main app

Fixed visual bug in sidenav, it had double scroll bars. Added overflow:hidden to remove one. (seen in firefox)
[Bug image](http://i.imgur.com/0KvBR3G.png)

EDIT:
overflow:hidden was responsible for hiding first item from list (science) and had to add margin-top: 64px (height of navbar) to fix it.

mentos1386 committed:
changed side nav, fixed double scroll bars

Fix: main app

Fixed visual bug in sidenav, it had double scroll bars. Added overflow:hidden to remove one. (seen in firefox)
[Bug image](http://i.imgur.com/0KvBR3G.png)

mentos1386 committed:
fixed sidenav, first item not visible

Fix: main app

margin-top set to 64px(height of navbar). First item in menu (science) was hidden under "All", margin pushed all down and it is in proper place now.

chutterbox commented:
@mentos1386 thanks for pointing this out. I see what you're talking about. There seems to be two issues: 

1) the first two(ish) networks are being cut off in firefox

2) double scroll bars

i think you're onto the right approach, but the toolbar shifts if there is scrollable content in the middle section, and i use transform: translatey to handle that interaction (see nav-controllers). 

the most immediate issue is that the #scrolly-left and #scrolly-right ids don't have the -moz-transform property in chutter.sass. (it has -webkit-transform and transform, but is missing -moz-transform), adding this back in would fix number 1)

the second issue may mean we need a little bit better solution, ideally this is the toolbar behavior we should be seeing: https://www.youtube.com/watch?v=eR9K1UKTe0c

mentos1386 committed:
improved over prevouse PR

fix: main app

Improved prevouse two PR

- added -moz-transform on scrolly-right and scrolly-left
- double scroll bars remved (without previuse margin)
- when scroling on main content works as intended (as on video)

chutterbox commented:
i think this is resovled/rendered obsolete by the new layout and what not. will reopen if we still need it
